client/src/pages/Signup.js

```diff
@@ -30,10 +31,18 @@ function Signup() {
         password: signUpCreds.password,
       })
       .then((response) => {
-        if (!response.data.error) {
+        console.log('response signup: ', response)
+        if (!response.data.error && !response.data.errors) {
+          console.log('signup success');
+          setErrorCode('');
+          alert(`${response.data.username} is now signed up, please log in`);
           history.replace('/Login');
+        } else if (response.data.error) {
+          setErrorCode('Please choose a different username');
```

Comment:
maybe add that the username is taken, or that the username has to be a valid email